docs/output.md

```diff
@@ -568,26 +585,24 @@ Currently, it contains:
 - _Protein_position_: Relative position of amino acid in protein
 - _BIOTYPE_: Biotype of transcript or regulatory feature
 
-For all samples:
+For further reading and documentation see the [VEP manual](https://www.ensembl.org/info/docs/tools/vep/index.html)
 
-**Output directory: `results/Annotation/[SAMPLE]/VEP`**
+_For all samples_:
```

Comment:
@maxulysse for the fields that are avaialbel in VEP now, I would need your help to make sure none are forgotten

Reply:
I'd add plus any additional fields selected via the plugins:
dbnsfp, loftee, spliceai, spliceregion